tutorials/kubernetes.md

```diff
@@ -2,40 +2,49 @@
 
 ## Switching runtime from docker to CRI-O
 
-In standard docker kubernetes cluster, kubelet is running on each node as systemd service and is taking care of communication between runtime and api service.
-It is reponsible for starting microservices pods (such as `kube-proxy`, `kubedns`, etc. - they can be different for various ways of deploying k8s) and user pods.
+In standard docker kubernetes cluster, kubelet is running on each node as systemd
+service and is taking care of communication between runtime and api service.
+It is reponsible for starting microservices pods (such as `kube-proxy`, `kubedns`,
+etc. - they can be different for various ways of deploying k8s) and user pods.
 Configuration of kubelet determines which runtime is used and in what way.
 
-Kubelet itself is executed in docker container (as we can see in `kubelet.service`), but, what is important, **it's not** a kubernetes pod (at least for now), 
-so we can keep kubelet running inside container (as well as directly on the host), and regardless of this, run pods in chosen runtime.
+Kubelet itself is executed in docker container (as we can see in `kubelet.service`),
+but, what is important, **it's not** a kubernetes pod (at least for now),
+so we can keep kubelet running inside container (as well as directly on the host),
+and regardless of this, run pods in chosen runtime.
 
-Below, you can find an instruction how to switch one or more nodes on running kubernetes cluster from docker to CRI-O.
+Below, you can find an instruction how to switch one or more nodes on running
+kubernetes cluster from docker to CRI-O.
 
 ### Preparing kubelet
+
 At first, you need to stop kubelet service working on the node:
+
+```shell
+systemctl stop kubelet
 ```
-# systemctl stop kubelet
-```
+
 and stop all kubelet docker containers that are still running.
 
-```
-# docker stop $(docker ps | grep k8s_ | awk '{print $1}')
+```shell
+docker stop $(docker ps | grep k8s_ | awk '{print $1}')
 ```
 
 We have to be sure that `kubelet.service` will start after `crio.service`.
 It can be done by adding `crio.service` to `Wants=` section in `/etc/systemd/system/kubelet.service`:
 
-```
-# cat /etc/systemd/system/kubelet.service | grep Wants
+```shell
+$ cat /etc/systemd/system/kubelet.service | grep Wants
 Wants=docker.socket crio.service
 ```
 
-If you'd like to change the way of starting kubelet (e.g. directly on host instead of docker container), you can change it here, but, as mentioned, it's not necessary.
-
+If you'd like to change the way of starting kubelet (e.g. directly on hostinstead
```

Comment:
```suggestion
If you'd like to change the way of starting kubelet (e.g., directly on the host instead
```

Reply:
I changed it to: (e.g. directly on the host instead of in a container)